www/astro.config.mjs

```diff
@@ -8,11 +8,10 @@ import remarkCodeTitles from "remark-code-titles";
 import rehypeSlug from "rehype-slug";
 import rehypeAutolinkHeadings from "rehype-autolink-headings";
 import sitemap from "@astrojs/sitemap";
-/**
- * @link https://astro.build/config
- */
+
+/** @link https://astro.build/config */
 export default defineConfig({
-  site: `https://create.t3.gg`,
+  site: `https://create.t3.gg/`,
```

Comment:
Per Astro upgrade guide: https://docs.astro.build/en/guides/upgrade-to/v2/#changed-astrosite-trailing-slash